crazyflie_sim/crazyflie_sim/simtypes.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+
```

Comment:
To me State and Action are the simulation data types. By types you were thinking about an enum instead? Would simdatatypes.py be better?

Reply:
ah yes that is how you can read it too.  Let's make it simdatatypes because I might not be the only one perhaps :) And for consistency, sim_data_types.py ?